### PR TITLE
added book controller & mapper, added findById functionality

### PR DESCRIPTION
**Pull request by BlueVioletTeti**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -83,6 +88,33 @@
...
+					<sourceDirectories>src</sourceDirectories>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
```
Comment (nacenik) [inline pom.xml]: Do you use Java 17 or Java 8? 
you can use property `java.version`

File: pom.xml [hunk 2]
```diff
@@ -83,6 +88,33 @@
...
+					<annotationProcessorPaths>
+						<path>
+							<groupId>org.mapstruct</groupId>
+							<artifactId>mapstruct-processor</artifactId>
+							<version>${org.mapstruct.version}</version>
+						</path>
+						<path>
+							<groupId>org.projectlombok</groupId>
+							<artifactId>lombok</artifactId>
+							<version>1.18.28</version>
```
Comment (nacenik) [inline pom.xml]: put this version as property

File: pom.xml [hunk 3]
```diff
@@ -83,6 +88,33 @@
...
+						</path>
+						<path>
+							<groupId>org.projectlombok</groupId>
+							<artifactId>lombok-mapstruct-binding</artifactId>
+							<version>0.2.0</version>
```
Comment (nacenik) [inline pom.xml]: the same.

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 4]
```diff
@@ -25,6 +25,5 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
     private String description;
-    @Column(name = "cover_image")
```
Comment (nacenik) [inline src/main/java/mate/academy/bookstore/model/Book.java:28]: why did you delete the column name?

**Review by nacenik — CHANGES_REQUESTED** [review]
Good solution, let's improve it :)

File: src/main/java/mate/academy/bookstore/model/Book.java (hunk 4 above)
Comment (BlueVioletTeti) [reply src/main/java/mate/academy/bookstore/model/Book.java:28]: I was told exactly the same column name would be parsed by sql without any annotations. Isn't it redundant code then?

Comment (vovasalyha) [reply src/main/java/mate/academy/bookstore/model/Book.java:28]: It was my comment that it is not required, because Hibernate does it by default, but it's if you do it explicitly, so it's for you to decide, there is no right answer

**Review by vovasalyha — APPROVED** [review]
